reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -826,6 +818,11 @@ static void sendDecodingFailures(
 							httpMessageLogFactory.warn(HttpMessageArgProviderFactory.create(msg)) : msg);
 		}
 
+		if (connectionInfo == null) {
```

Comment:
`connectionInfo` is necessary only when we create `FailedHttpServerRequest`, isn't it?

Then at that point we have the original `request`

Why are checking for http/2?
`isHttp2 ? ctx.channel().parent() : ctx.channel()`

Reply:
> Why are we checking for http/2?
> isHttp2 ? ctx.channel().parent() : ctx.channel()

I just removed this useless check in commit https://github.com/reactor/reactor-netty/pull/2755/commits/e34cfc22d4b0e56b24ab05efb6c1eebdba13d4cd, sorry.